spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
As a side note. We also need to check `Partners` and `Sponsors` tests. May they have the same problem.